### PR TITLE
Feat/120 shelter project comparison

### PR DESCRIPTION
**Pull request by guilbep**
# Describe your changes

See issue to have a pdf/image of the result of this implemtentaion

# Related GitHub issues and pull requests

- Ref: #120

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] Don't forget to link PR to issue if you are solving one.

### CODE REVIEW

File: frontend/src/store/ShelterInterface.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ export const listOfShelterType: ShelterType[] = [
   "Durable",
 ];
 
-export type ShelterType = "Emergency" | "Transitional" | "Durable";
+export type ShelterType = "Emergency" | "Transitional" | "Durable" | "";
```
Comment (blueur) [inline frontend/src/store/ShelterInterface.ts:20]: empty ?! why not undefined or null ?

File: frontend/src/store/ShelterModule.ts [hunk 2]
```diff
@@ -97,16 +98,21 @@ const actions: ActionTree<ShelterState, RootState> = {
   },
   getScorecards: (
     context: ActionContext<ShelterState, RootState>,
-    id: string
+    ids: string | string[]
   ) => {
     const localCouch = context.state.localCouch;
     // shelters/_design/shelter/_view/scorecards?include_docs=true
+    const selectedIds = isString(ids) ? [ids] : ids;
```
Comment (blueur) [inline frontend/src/store/ShelterModule.ts:105]: typeof ids === "string" ?

File: frontend/src/views/shelter_sustainability/ShelterSustainabilityItem/affordabilities.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { ScorecardConfig } from "@/views/shelter_sustainability/ShelterSustainabilityItem/generateScorecardOptions";
+import { alpha } from "@/views/shelter_sustainability/shelterTypeColors";
+
+export const affordabilities: ScorecardConfig[] = [
+  {
+    id: "affordability",
+    title: "Affordability",
+    min: 0,
+    // max: 100,
```
Comment (blueur) [inline frontend/src/views/shelter_sustainability/ShelterSustainabilityItem/affordabilities.ts:9]: dead code ?

File: frontend/src/views/shelter_sustainability/ShelterSustainabilityItem/affordabilities.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    unit: "$/year/m²",
+    description:
+      "Affordability score describes shelter cost (USD) per year (of intended use) per square meter (of habitable space), enabling comparison across shelters of differing size and durability.",
+    colors: {
+      primary: `rgba(248, 228, 210, 1)`, // seconday colour 5
```
Comment (blueur) [inline frontend/src/views/shelter_sustainability/ShelterSustainabilityItem/affordabilities.ts:14]: color in consts ?

File: frontend/src/views/shelter_sustainability/ShelterSustainabilityList.vue [hunk 5]
```diff
@@ -422,6 +464,11 @@ export default class ProjectList extends Vue {
 
     // reload on db change
     this.db?.onChange(this.getShelters);
+
+    const queryIds = this.$route.query.ids;
+    if (isString(queryIds)) {
```
Comment (blueur) [inline frontend/src/views/shelter_sustainability/ShelterSustainabilityList.vue:469]: typeof queryIds === "string"

File: frontend/src/components/shelter_sustainability/billOfQuantities/graphTree.vue [hunk 6]
```diff
@@ -230,15 +236,15 @@ export default class GraphTree extends Vue {
...
+          radius: this.radius,
           nodeClick: undefined,
           data: this.generateTreeData,
           animationDurationUpdate: 1000,
           universalTransition: true,
           itemStyle: {
             borderWidth: 1,
             borderColor: "rgba(150,160,255,.5)",
-            borderRadius: 7,
+            // borderRadius: 7,
```
[the remark this answers is not in the document]
Comment (guilbep) [reply frontend/src/components/shelter_sustainability/billOfQuantities/graphTree.vue]: Thanks

File: frontend/src/plugins/vuetify.ts [hunk 7]
```diff
@@ -166,15 +175,19 @@ export default new Vuetify({
  * https://data2.unhcr.org/en/documents/download/60115
  * // https://unhcr-web.github.io/unhcRstyle/docs/reference/unhcr_green.html
  */
+
+function colorTemplate(literals: TemplateStringsArray, alpha = "1") {
```
Comment (guilbep) [inline frontend/src/plugins/vuetify.ts:179]: @blueur I changed the colors so that it's defined only in one place

**Pull request merged**